matrix_content_scanner/utils/constants.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2022 The Matrix.org Foundation C.I.C.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from enum import Enum
+
+
+class ErrCode(Enum):
+    def __str__(self) -> str:
+        return str(self.value)
+
+    def __repr__(self) -> str:
+        return str(self.value)
+
+    # An unknown error happened.
+    UNKNOWN = "M_UNKNOWN"
```

Comment:
These ErrCodes don't seem to be used in any tests.
It might be that you've deferred that to a later chunk of work (fine), otherwise it would be worth having some tests check that these do what you expect

Reply:
Yeah that (and some other codes) is used in other bits of code. This one specifically is used to respond to the user when a servlet raised an unknown error: https://github.com/matrix-org/matrix-content-scanner-python-wip/blob/a1396204375732f31896df5f9360364a39fbb6b5/matrix_content_scanner/servlets/__init__.py#L83-L91